eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -75,7 +75,7 @@ jobs:
 
       - template: eng/pipelines/templates/scripts/replace-relative-links.yml@azure-sdk-tools
         parameters:
-          TargetFolder: $(Build.SourcesDirectory)/sdk/$(folder)
+          TargetFolder: $(Build.SourcesDirectory)
```

Comment:
I don't think this is the correct fix. Instead the fix should be to add this step to to doc index pipeline, similar to what JIm did in java https://github.com/Azure/azure-sdk-for-java/pull/6925

Reply:
done